rules/windows/powershell/powershell_script/posh_ps_tamper_defender.yml

```
@@ -52,7 +52,22 @@ detection:
             - LowThreatDefaultAction
             - ModerateThreatDefaultAction
             - HighThreatDefaultAction
-    condition: all of selection_options_disabling* or selection_default_actions_allow
+    selection_use_of_alias:
+        ScriptBlockText|contains:
+            - ltdefac
+            - mtdefac
+            - htdefac
+            - stdefac
+    filter_alias:
+        # powershell use of Get-MpComputerStatus loads ps1 for management
+        # Host Application = C:\Windows\System32\WindowsPowershell\v1.0\powershell.exe -command Get-MpComputerStatus Engine Version = 5.1.17763.3770 Runspace ID = 86662fa7-ad6f-4ead-93a2-267be387d004 Pipeline ID = 6 Command Name = Remove-Variable Command Type = Cmdlet Script Name = Command Path = Sequence Number = 1825 User = HUNT\hawkscan Connected User = Shell ID = Microsoft.PowerShell | Payload=CommandInvocation(Remove-Variable): "Remove-Variable" ParameterBinding(Remove-Variable): name="Scope"; value="local" ParameterBinding(Remove-Variable): name="Confirm"; value="False" ParameterBinding(Remove-Variable): name="Name"; value="PreviousErrCount"
+        ScriptBlockText|contains|all:
+            - "Alias('ltdefac')"
+            - "Alias('mtdefac')"
+            - "Alias('htdefac')"
+            - "Alias('stdefac')"
```

I would write it like this:
```suggestion
    selection_use_of_alias:
        ScriptBlockText|contains:
            - 'ltdefac '
            - 'mtdefac '
            - 'htdefac '
            - 'stdefac '
```